test/stat_tracker_test.rb

```diff
@@ -191,12 +191,43 @@ def test_shots_to_goals_ratio_per_team
   end
 
   def test_it_can_return_games_won_per_team_for_a_season
-    assert_equal ({"6"=>9, "16"=>1}), @@stat_tracker.games_won_per_team_for("20122013")
+    assert_equal ({"6"=>9, "16"=>1}), @@stat_tracker.games_won_per_team_for_season("20122013")
   end
 
   def test_it_can_return_total_games_per_team_for_a_season
-    assert_equal ({"3"=>5, "6"=>9, "5"=>4, "17"=>1, "16"=>1}), @@stat_tracker.total_games_per_team_for("20122013")
+    assert_equal ({"3"=>5, "6"=>9, "5"=>4, "17"=>1, "16"=>1}), @@stat_tracker.total_games_per_team_for_season("20122013")
+  end
+
+  def test_it_can_return_all_coaches
+    expected = ["John Tortorella", "Claude Julien", "Dan Bylsma", "Mike Babcock", "Joel Quenneville", "Jon Cooper", "Mike Johnston", "Alain Vigneault", "Peter DeBoer", "Gerard Gallant", "Randy Carlyle"]
+   assert_equal expected, @@stat_tracker.all_coaches
+  end
+
+  def test_it_can_get_team_games_by_coach
+    assert_instance_of Hash, @@stat_tracker.create_hash_with_team_games_by_coach("20122013")
+    assert_equal 11, @@stat_tracker.create_hash_with_team_games_by_coach("20122013").length
+  end
+
+  def test_it_can_get_games_by_coach_and_season
+    assert_instance_of Array, @@stat_tracker.game_teams_that_season_by_coach("Mike Babcock", "20122013")
+    assert_equal 9, @@stat_tracker.game_teams_that_season_by_coach("Claude Julien", "20122013").length
+  end
```

Comment:
What does everyone think about the documentation portion of this? Should this test be refactored or left as is? @sarynm12 @hopesgit @leahriffell

Reply:
Are you saying because we aren't asserting to any specific values (just the class and size of collection)?

Reply:
Yeah! 

Not asserting specific values is probably okay as far as testing goes. But for documentation, when I look at this test I can't fully tell what this method is returning. 

If we explicitly build out an "expected" value then you could see exactly what to expect from this method just at a glance. 

This is nit-picky though! So I'm okay either way.

Reply:
Yeah I get what you're saying. It would be easier to understand what's being returned here if there was an "expected" in the test. 
Saryn it looks like maybe you didn't add expected for the Hash test b/c you're adding in instances of game_team and you don't have variable names to assert to... is that correct?

Reply:
Yes, that's correct! But I totally see what you guys are saying! The expected ends up being pretty long, but I'm okay with doing it if that's what we decide! Or I can re-name the test to make it a little clearer. Sorry for the confusion on this!